File: 2278-FindThreeConsecutiveIntegersThatSumToAGivenNumber/2278-FindThreeConsecutiveIntegersThatSumToAGivenNumber.py
# Last updated: 11/05/2026, 01:31:04
class Solution:
    def sumOfThree(self, num: int) -> List[int]:
        l = []
        if (num % 3 == 0):
            l.append(num//3-1)
            l.append(num//3)
            l.append(num//3+1)
        return l
            

File: 2278-FindThreeConsecutiveIntegersThatSumToAGivenNumber/2278-FindThreeConsecutiveIntegersThatSumToAGivenNumber.cpp
// Last updated: 11/05/2026, 01:31:16
class Solution {
public:
    vector<long long> sumOfThree(long long num) {
        vector<long long> n;
        if (num%3 == 0){
            long long p = num/3;
            n.push_back(p-1);
            n.push_back(p);
            n.push_back(p+1);
        }
        return n;
    }
};